soc/xtensa/nxp_adsp/imx8/include/soc/memory.h

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2021 NXP Semiconductors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_MEMORY_H
+#define __INC_MEMORY_H
+
+#define PLATFORM_CORE_COUNT 1
+
+/** Id of master DSP core */
+#define PLATFORM_PRIMARY_CORE_ID	0
+
+#define SRAM_RESERVE_HEADER_SPACE	0x00010000
+
+#define SRAM_BASE	0x596E8400
+#define SRAM_SIZE	0x880000
```

Comment:
Some of this looks like it might be copied over from the intel_adsp code?  Note that that code comes directly from SOF.  For new platforms, the Zephyr model is to put most intelligence like this about memory layout and register addresses into devicetree files.

Reply:
These two, SRAM_BASE and SRAM_SIZE, are never used for iMX.
The address and the size are correct, but the naming is incorrect.

But, since they are not used, I'll remove them.

Reply:
I've removed the unused variables from imx8, memory.h file.